sources/constraint-declaration-validation.asciidoc

```diff
@@ -878,7 +878,7 @@ public interface ParameterNameProvider {
 
 [tck-testable]#A conforming Bean Validation implementation provides a default [classname]`ParameterNameProvider` implementation which returns parameter names in the form arg``PARAMETER_INDEX``, where `PARAMETER_INDEX` starts at 0 for the first parameter, e.g. [varname]`arg0`, [varname]`arg1` etc.#
 
-Bean Validation providers and integrators are free to provide additional implementations (e.g. based on annotations specifying parameter names, debug symbols etc.). If a user wishes to use another parameter name provider than the default implementation, she may specify the provider to use with help of the bootstrap API (see <<bootstrapping>>) or the XML configuration (see <<xml-config>>).
+Bean Validation providers and integrators are free to provide additional implementations (e.g. based on annotations specifying parameter names, debug symbols etc.). If a user wishes to use another parameter name provider than the default implementation, he may specify the provider to use with help of the bootstrap API (see <<bootstrapping>>) or the XML configuration (see <<xml-config>>).
```

Comment:
Ok, let's not get hung up about this one. Keep the change it if you feel
strongly about it, just meant to clarify it wasn't an oversight in the
original version. I tend to use the "singular they" mostly these days, so
that'd be my preference if we want to change it at all.